plugin.json

```diff
@@ -7,9 +7,10 @@
     "license": "proprietary",
     "link": "http://store.shopware.com",
     "author": "Shopware AG",
-    "currentVersion": "1.1.14",
+    "currentVersion": "1.1.15",
     "changelog": {
         "de": {
+            "1.1.15": "CON-5085 - Templates werden jetzt früher im Dispatchprozess registriert",
```

Comment:
yep that s the technical description but users don't understand this 
i would recommend to change this to something like
`Added: Compatibility for SmartySecurity`